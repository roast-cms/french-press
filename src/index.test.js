import "jest-styled-components"

import React from "react"

import {shallow} from "enzyme"

import {FrenchPress} from "./"

test("Render FrenchPress without crashing, matches snapshot", () => {
  const element = shallow(<FrenchPress />)
  expect(element).toMatchSnapshot()
})
