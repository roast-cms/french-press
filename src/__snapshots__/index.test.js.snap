// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FrenchPress />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": "Editor",
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <_default
          click={[Function]}
          style={
            Object {
              "display": "none",
              "opacity": "0",
              "top": 0,
            }
          }
        >
          <Unknown />
        </_default>,
        <Editor
          autoCorrect={true}
          autoFocus={false}
          formatCommand={[Function]}
          onChange={[Function]}
          onClick={[Function]}
          options={
            Object {
              "domain": "",
              "imagePlaceholder": "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
            }
          }
          placeholder="Write your storyâ€¦"
          plugins={
            Array [
              Object {
                "onPaste": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onDrop": [Function],
                "onPaste": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
            ]
          }
          readOnly={false}
          renderMark={[Function]}
          renderNode={[Function]}
          schema={
            Object {
              "blocks": Object {
                "divider": Object {
                  "isVoid": true,
                },
                "docket": Object {
                  "isVoid": true,
                },
                "image": Object {
                  "data": Object {
                    "src": [Function],
                  },
                  "isVoid": true,
                  "normalize": [Function],
                  "parent": Array [
                    Object {
                      "object": "document",
                    },
                  ],
                },
                "link": Object {
                  "nodes": Array [
                    Object {
                      "objects": Array [
                        "text",
                      ],
                    },
                  ],
                },
              },
              "document": Object {
                "last": Object {
                  "types": Array [
                    "paragraph",
                    "quote",
                  ],
                },
                "nodes": Array [
                  Object {
                    "types": Array [
                      "paragraph",
                      "heading",
                      "divider",
                      "quote",
                      "image",
                      "docket",
                      "link",
                    ],
                  },
                ],
                "normalize": [Function],
              },
              "inlines": Object {
                "heading": Object {
                  "nodes": Array [
                    Object {
                      "types": Array [
                        "text",
                      ],
                    },
                  ],
                },
                "paragraph": Object {
                  "nodes": Array [
                    Object {
                      "types": Array [
                        "text",
                        "link",
                      ],
                    },
                  ],
                },
                "quote": Object {
                  "nodes": Array [
                    Object {
                      "types": Array [
                        "text",
                      ],
                    },
                  ],
                },
              },
            }
          }
          spellCheck={true}
          style={
            Object {
              "background": "",
              "boxShadow": "",
              "minHeight": "28em",
            }
          }
          value={
            Immutable.Record {
              "data": Immutable.Map {},
              "decorations": null,
              "document": Immutable.Record {
                "data": Immutable.Map {},
                "key": "1",
                "nodes": Immutable.List [
                  Immutable.Record {
                    "data": Immutable.Map {},
                    "isVoid": false,
                    "key": "2",
                    "nodes": Immutable.List [
                      Immutable.Record {
                        "leaves": Immutable.List [
                          Immutable.Record {
                            "marks": Immutable.Set [],
                            "text": "",
                          },
                        ],
                        "key": "3",
                      },
                    ],
                    "type": "paragraph",
                  },
                ],
              },
              "history": Immutable.Record {
                "redos": Immutable.Stack [],
                "undos": Immutable.Stack [],
              },
              "schema": Immutable.Record {
                "stack": Immutable.Record {
                  "plugins": Array [
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "schema": Object {},
                    },
                  ],
                },
                "document": Object {
                  "data": Object {},
                  "nodes": null,
                },
                "blocks": Object {},
                "inlines": Object {},
              },
              "selection": Immutable.Record {
                "anchorKey": "3",
                "anchorOffset": 0,
                "anchorPath": Immutable.List [
                  0,
                  0,
                ],
                "focusKey": "3",
                "focusOffset": 0,
                "focusPath": Immutable.List [
                  0,
                  0,
                ],
                "isAtomic": false,
                "isBackward": false,
                "isFocused": false,
                "marks": null,
              },
            }
          }
        />,
        <input
          accept="image/png,image/jpeg"
          onChange={[Function]}
          style={
            Object {
              "display": "none",
            }
          }
          type="file"
        />,
      ],
      "style": Object {
        "position": "relative",
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <Unknown />,
          "click": [Function],
          "style": Object {
            "display": "none",
            "opacity": "0",
            "top": 0,
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "autoCorrect": true,
          "autoFocus": false,
          "callbackError": undefined,
          "components": undefined,
          "controls": undefined,
          "fileInputRef": undefined,
          "formatCommand": [Function],
          "onChange": [Function],
          "onClick": [Function],
          "options": Object {
            "domain": "",
            "imagePlaceholder": "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
          },
          "placeholder": "Write your storyâ€¦",
          "plugins": Array [
            Object {
              "onPaste": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
            Object {
              "onDrop": [Function],
              "onPaste": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
            Object {
              "onKeyDown": [Function],
            },
          ],
          "readOnly": false,
          "renderMark": [Function],
          "renderNode": [Function],
          "schema": Object {
            "blocks": Object {
              "divider": Object {
                "isVoid": true,
              },
              "docket": Object {
                "isVoid": true,
              },
              "image": Object {
                "data": Object {
                  "src": [Function],
                },
                "isVoid": true,
                "normalize": [Function],
                "parent": Array [
                  Object {
                    "object": "document",
                  },
                ],
              },
              "link": Object {
                "nodes": Array [
                  Object {
                    "objects": Array [
                      "text",
                    ],
                  },
                ],
              },
            },
            "document": Object {
              "last": Object {
                "types": Array [
                  "paragraph",
                  "quote",
                ],
              },
              "nodes": Array [
                Object {
                  "types": Array [
                    "paragraph",
                    "heading",
                    "divider",
                    "quote",
                    "image",
                    "docket",
                    "link",
                  ],
                },
              ],
              "normalize": [Function],
            },
            "inlines": Object {
              "heading": Object {
                "nodes": Array [
                  Object {
                    "types": Array [
                      "text",
                    ],
                  },
                ],
              },
              "paragraph": Object {
                "nodes": Array [
                  Object {
                    "types": Array [
                      "text",
                      "link",
                    ],
                  },
                ],
              },
              "quote": Object {
                "nodes": Array [
                  Object {
                    "types": Array [
                      "text",
                    ],
                  },
                ],
              },
            },
          },
          "spellCheck": true,
          "style": Object {
            "background": "",
            "boxShadow": "",
            "minHeight": "28em",
          },
          "value": Immutable.Record {
            "data": Immutable.Map {},
            "decorations": null,
            "document": Immutable.Record {
              "data": Immutable.Map {},
              "key": "1",
              "nodes": Immutable.List [
                Immutable.Record {
                  "data": Immutable.Map {},
                  "isVoid": false,
                  "key": "2",
                  "nodes": Immutable.List [
                    Immutable.Record {
                      "leaves": Immutable.List [
                        Immutable.Record {
                          "marks": Immutable.Set [],
                          "text": "",
                        },
                      ],
                      "key": "3",
                    },
                  ],
                  "type": "paragraph",
                },
              ],
            },
            "history": Immutable.Record {
              "redos": Immutable.Stack [],
              "undos": Immutable.Stack [],
            },
            "schema": Immutable.Record {
              "stack": Immutable.Record {
                "plugins": Array [
                  Object {
                    "validateNode": [Function],
                  },
                  Object {
                    "validateNode": [Function],
                  },
                  Object {
                    "validateNode": [Function],
                  },
                  Object {
                    "validateNode": [Function],
                  },
                  Object {
                    "validateNode": [Function],
                  },
                  Object {
                    "validateNode": [Function],
                  },
                  Object {
                    "validateNode": [Function],
                  },
                  Object {
                    "validateNode": [Function],
                  },
                  Object {
                    "schema": Object {},
                  },
                ],
              },
              "document": Object {
                "data": Object {},
                "nodes": null,
              },
              "blocks": Object {},
              "inlines": Object {},
            },
            "selection": Immutable.Record {
              "anchorKey": "3",
              "anchorOffset": 0,
              "anchorPath": Immutable.List [
                0,
                0,
              ],
              "focusKey": "3",
              "focusOffset": 0,
              "focusPath": Immutable.List [
                0,
                0,
              ],
              "isAtomic": false,
              "isBackward": false,
              "isFocused": false,
              "marks": null,
            },
          },
        },
        "ref": [Function],
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "accept": "image/png,image/jpeg",
          "onChange": [Function],
          "style": Object {
            "display": "none",
          },
          "type": "file",
        },
        "ref": [Function],
        "rendered": null,
        "type": "input",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": "Editor",
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <_default
            click={[Function]}
            style={
              Object {
                "display": "none",
                "opacity": "0",
                "top": 0,
              }
            }
          >
            <Unknown />
          </_default>,
          <Editor
            autoCorrect={true}
            autoFocus={false}
            formatCommand={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            options={
              Object {
                "domain": "",
                "imagePlaceholder": "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
              }
            }
            placeholder="Write your storyâ€¦"
            plugins={
              Array [
                Object {
                  "onPaste": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
                Object {
                  "onDrop": [Function],
                  "onPaste": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
                Object {
                  "onKeyDown": [Function],
                },
              ]
            }
            readOnly={false}
            renderMark={[Function]}
            renderNode={[Function]}
            schema={
              Object {
                "blocks": Object {
                  "divider": Object {
                    "isVoid": true,
                  },
                  "docket": Object {
                    "isVoid": true,
                  },
                  "image": Object {
                    "data": Object {
                      "src": [Function],
                    },
                    "isVoid": true,
                    "normalize": [Function],
                    "parent": Array [
                      Object {
                        "object": "document",
                      },
                    ],
                  },
                  "link": Object {
                    "nodes": Array [
                      Object {
                        "objects": Array [
                          "text",
                        ],
                      },
                    ],
                  },
                },
                "document": Object {
                  "last": Object {
                    "types": Array [
                      "paragraph",
                      "quote",
                    ],
                  },
                  "nodes": Array [
                    Object {
                      "types": Array [
                        "paragraph",
                        "heading",
                        "divider",
                        "quote",
                        "image",
                        "docket",
                        "link",
                      ],
                    },
                  ],
                  "normalize": [Function],
                },
                "inlines": Object {
                  "heading": Object {
                    "nodes": Array [
                      Object {
                        "types": Array [
                          "text",
                        ],
                      },
                    ],
                  },
                  "paragraph": Object {
                    "nodes": Array [
                      Object {
                        "types": Array [
                          "text",
                          "link",
                        ],
                      },
                    ],
                  },
                  "quote": Object {
                    "nodes": Array [
                      Object {
                        "types": Array [
                          "text",
                        ],
                      },
                    ],
                  },
                },
              }
            }
            spellCheck={true}
            style={
              Object {
                "background": "",
                "boxShadow": "",
                "minHeight": "28em",
              }
            }
            value={
              Immutable.Record {
                "data": Immutable.Map {},
                "decorations": null,
                "document": Immutable.Record {
                  "data": Immutable.Map {},
                  "key": "1",
                  "nodes": Immutable.List [
                    Immutable.Record {
                      "data": Immutable.Map {},
                      "isVoid": false,
                      "key": "2",
                      "nodes": Immutable.List [
                        Immutable.Record {
                          "leaves": Immutable.List [
                            Immutable.Record {
                              "marks": Immutable.Set [],
                              "text": "",
                            },
                          ],
                          "key": "3",
                        },
                      ],
                      "type": "paragraph",
                    },
                  ],
                },
                "history": Immutable.Record {
                  "redos": Immutable.Stack [],
                  "undos": Immutable.Stack [],
                },
                "schema": Immutable.Record {
                  "stack": Immutable.Record {
                    "plugins": Array [
                      Object {
                        "validateNode": [Function],
                      },
                      Object {
                        "validateNode": [Function],
                      },
                      Object {
                        "validateNode": [Function],
                      },
                      Object {
                        "validateNode": [Function],
                      },
                      Object {
                        "validateNode": [Function],
                      },
                      Object {
                        "validateNode": [Function],
                      },
                      Object {
                        "validateNode": [Function],
                      },
                      Object {
                        "validateNode": [Function],
                      },
                      Object {
                        "schema": Object {},
                      },
                    ],
                  },
                  "document": Object {
                    "data": Object {},
                    "nodes": null,
                  },
                  "blocks": Object {},
                  "inlines": Object {},
                },
                "selection": Immutable.Record {
                  "anchorKey": "3",
                  "anchorOffset": 0,
                  "anchorPath": Immutable.List [
                    0,
                    0,
                  ],
                  "focusKey": "3",
                  "focusOffset": 0,
                  "focusPath": Immutable.List [
                    0,
                    0,
                  ],
                  "isAtomic": false,
                  "isBackward": false,
                  "isFocused": false,
                  "marks": null,
                },
              }
            }
          />,
          <input
            accept="image/png,image/jpeg"
            onChange={[Function]}
            style={
              Object {
                "display": "none",
              }
            }
            type="file"
          />,
        ],
        "style": Object {
          "position": "relative",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <Unknown />,
            "click": [Function],
            "style": Object {
              "display": "none",
              "opacity": "0",
              "top": 0,
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "autoCorrect": true,
            "autoFocus": false,
            "callbackError": undefined,
            "components": undefined,
            "controls": undefined,
            "fileInputRef": undefined,
            "formatCommand": [Function],
            "onChange": [Function],
            "onClick": [Function],
            "options": Object {
              "domain": "",
              "imagePlaceholder": "data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7",
            },
            "placeholder": "Write your storyâ€¦",
            "plugins": Array [
              Object {
                "onPaste": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onDrop": [Function],
                "onPaste": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
              Object {
                "onKeyDown": [Function],
              },
            ],
            "readOnly": false,
            "renderMark": [Function],
            "renderNode": [Function],
            "schema": Object {
              "blocks": Object {
                "divider": Object {
                  "isVoid": true,
                },
                "docket": Object {
                  "isVoid": true,
                },
                "image": Object {
                  "data": Object {
                    "src": [Function],
                  },
                  "isVoid": true,
                  "normalize": [Function],
                  "parent": Array [
                    Object {
                      "object": "document",
                    },
                  ],
                },
                "link": Object {
                  "nodes": Array [
                    Object {
                      "objects": Array [
                        "text",
                      ],
                    },
                  ],
                },
              },
              "document": Object {
                "last": Object {
                  "types": Array [
                    "paragraph",
                    "quote",
                  ],
                },
                "nodes": Array [
                  Object {
                    "types": Array [
                      "paragraph",
                      "heading",
                      "divider",
                      "quote",
                      "image",
                      "docket",
                      "link",
                    ],
                  },
                ],
                "normalize": [Function],
              },
              "inlines": Object {
                "heading": Object {
                  "nodes": Array [
                    Object {
                      "types": Array [
                        "text",
                      ],
                    },
                  ],
                },
                "paragraph": Object {
                  "nodes": Array [
                    Object {
                      "types": Array [
                        "text",
                        "link",
                      ],
                    },
                  ],
                },
                "quote": Object {
                  "nodes": Array [
                    Object {
                      "types": Array [
                        "text",
                      ],
                    },
                  ],
                },
              },
            },
            "spellCheck": true,
            "style": Object {
              "background": "",
              "boxShadow": "",
              "minHeight": "28em",
            },
            "value": Immutable.Record {
              "data": Immutable.Map {},
              "decorations": null,
              "document": Immutable.Record {
                "data": Immutable.Map {},
                "key": "1",
                "nodes": Immutable.List [
                  Immutable.Record {
                    "data": Immutable.Map {},
                    "isVoid": false,
                    "key": "2",
                    "nodes": Immutable.List [
                      Immutable.Record {
                        "leaves": Immutable.List [
                          Immutable.Record {
                            "marks": Immutable.Set [],
                            "text": "",
                          },
                        ],
                        "key": "3",
                      },
                    ],
                    "type": "paragraph",
                  },
                ],
              },
              "history": Immutable.Record {
                "redos": Immutable.Stack [],
                "undos": Immutable.Stack [],
              },
              "schema": Immutable.Record {
                "stack": Immutable.Record {
                  "plugins": Array [
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "validateNode": [Function],
                    },
                    Object {
                      "schema": Object {},
                    },
                  ],
                },
                "document": Object {
                  "data": Object {},
                  "nodes": null,
                },
                "blocks": Object {},
                "inlines": Object {},
              },
              "selection": Immutable.Record {
                "anchorKey": "3",
                "anchorOffset": 0,
                "anchorPath": Immutable.List [
                  0,
                  0,
                ],
                "focusKey": "3",
                "focusOffset": 0,
                "focusPath": Immutable.List [
                  0,
                  0,
                ],
                "isAtomic": false,
                "isBackward": false,
                "isFocused": false,
                "marks": null,
              },
            },
          },
          "ref": [Function],
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "accept": "image/png,image/jpeg",
            "onChange": [Function],
            "style": Object {
              "display": "none",
            },
            "type": "file",
          },
          "ref": [Function],
          "rendered": null,
          "type": "input",
        },
      ],
      "type": "div",
    },
    Object {
      "instance": null,
      "key": "Menu",
      "nodeType": "function",
      "props": Object {
        "controls": undefined,
        "formatCommand": [Function],
        "menuRef": [Function],
        "onChange": [Function],
        "style": Object {
          "display": "none",
        },
        "value": Immutable.Record {
          "data": Immutable.Map {},
          "decorations": null,
          "document": Immutable.Record {
            "data": Immutable.Map {},
            "key": "1",
            "nodes": Immutable.List [
              Immutable.Record {
                "data": Immutable.Map {},
                "isVoid": false,
                "key": "2",
                "nodes": Immutable.List [
                  Immutable.Record {
                    "leaves": Immutable.List [
                      Immutable.Record {
                        "marks": Immutable.Set [],
                        "text": "",
                      },
                    ],
                    "key": "3",
                  },
                ],
                "type": "paragraph",
              },
            ],
          },
          "history": Immutable.Record {
            "redos": Immutable.Stack [],
            "undos": Immutable.Stack [],
          },
          "schema": Immutable.Record {
            "stack": Immutable.Record {
              "plugins": Array [
                Object {
                  "validateNode": [Function],
                },
                Object {
                  "validateNode": [Function],
                },
                Object {
                  "validateNode": [Function],
                },
                Object {
                  "validateNode": [Function],
                },
                Object {
                  "validateNode": [Function],
                },
                Object {
                  "validateNode": [Function],
                },
                Object {
                  "validateNode": [Function],
                },
                Object {
                  "validateNode": [Function],
                },
                Object {
                  "schema": Object {},
                },
              ],
            },
            "document": Object {
              "data": Object {},
              "nodes": null,
            },
            "blocks": Object {},
            "inlines": Object {},
          },
          "selection": Immutable.Record {
            "anchorKey": "3",
            "anchorOffset": 0,
            "anchorPath": Immutable.List [
              0,
              0,
            ],
            "focusKey": "3",
            "focusOffset": 0,
            "focusPath": Immutable.List [
              0,
              0,
            ],
            "isAtomic": false,
            "isBackward": false,
            "isFocused": false,
            "marks": null,
          },
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
